project/frontend/src/index.js

```diff
@@ -21,10 +21,9 @@ ReactDOM.render(
       <Router>
         <Header />
         <Switch>
-          <Route exact path='/' component={HomePage} />
-          <Route path='/find-match' component={PreferenceForm} />
```

Comment:
Why did we get rid of the preference form route?

Reply:
Didn't need it anymore since `PreferenceForm` is now a popup over the `HomePage` component.